app/services/analytics_events.rb

```diff
@@ -2905,8 +2915,13 @@ def user_registration_suggest_another_mfa_notice_skipped
   end
 
   # Tracks when user visits MFA selection page
-  def user_registration_2fa_setup_visit
-    track_event('User Registration: 2FA Setup visited')
+  # @param [String] sign_up_mfa_priority_bucket
+  def user_registration_2fa_setup_visit(sign_up_mfa_priority_bucket:, **_extra)
+    track_event(
+      'User Registration: 2FA Setup visited', {
+        sign_up_mfa_priority_bucket: sign_up_mfa_priority_bucket,
+      }
```

Comment:
Same notes about dropping curly braces and logging `extra`.

```suggestion
  def user_registration_2fa_setup_visit(sign_up_mfa_priority_bucket:, **extra)
    track_event(
      'User Registration: 2FA Setup visited',
      sign_up_mfa_priority_bucket: sign_up_mfa_priority_bucket,
      **extra,
```